cross/mingw-w64-wine-mono/Portfile

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                mingw-w64-wine-mono
+categories          cross
+maintainers         {@gcenx gmail.com:gcenx83}
+homepage            https://github.com/madewokherd/wine-mono
+license             GPL, LGPL2.1, MPL
```

Comment:
this doesn't look correct or at least inconsistent with ysafe in other Portfiles. Please use just spaces if multiple licenses apply; if there is a choice add curly brackets. See the [guide](https://guide.macports.org/index.html#reference.keywords) for more details.

Reply:
thanks!
In this port as well: please correct your GitHub handle to have it match with what you're actually using when submitting a PR.